doc/source/user_guide/timeseries.rst

```diff
@@ -590,24 +595,28 @@ This starts on the very first time in the month, and includes the last date and
 time for the month:
 
 .. ipython:: python
+   :okwarning:
 
    dft['2013-1':'2013-2']
```

Comment:
This is not actually deprecated, right? (and thus shouldn't raise a warning?)

Reply:
Good catch.  I think this should be deprecated too.